### PR TITLE
ZOSVPC - 2871 Implementation of giving an option of creating z/os vsi with a specified profile size for quick start variation.

### PR DESCRIPTION
**Pull request by Nikhil22February**
Give the option of creating a z/OS VSI
with a specified profile size
validate the input profile size to ensure that z/OS VSI will IPL successfully - that is, >= 2 vCPUS and >= 16GB RAM. For example, the following existing IBM Z profiles will NOT work:
bz2-1x4
bz2-2x8
cz2-2x4
cz2-4x8
Support the profile size option for both quick start and standard variations

### CODE REVIEW

File: solutions/quickstart/README.md [hunk 1]
```diff
@@ -49,4 +49,4 @@ To view your current configuration, run the `terraform refresh` command.
 1. Export API Key
     export TF_VAR_ibmcloud_api_key=xxxx
 2. Run Terraform init
-3. Run Terraform apply
+3. Run Terraform apply
```
Comment (kevinkuhner) [inline solutions/quickstart/README.md:52]: You didn't make a change to the README but you must have changed something for it to show up in the PR.  Please checkout the original README again so it doesn't show under the PR.

File: solutions/quickstart/variables.tf [hunk 2]
```diff
@@ -277,6 +290,24 @@ variable "override_json_string" {
                            "port_min": 22
                          }
               },
+	      {
+                  "direction": "inbound",
+                  "name": "allow-all-inbound-1",
+                  "source": "0.0.0.0/0",
+                  "tcp": {
+                           "port_max": 992,
+                           "port_min": 992
```
Comment (kevinkuhner) [inline solutions/quickstart/variables.tf:299]: Why this change?   Its not related to your PR.

**Review by kevinkuhner — CHANGES_REQUESTED** [review]
@Nikhil22February  - please see my comments.

File: solutions/quickstart/variables.tf [hunk 3]
```diff
@@ -11,6 +11,9 @@ variable "ibmcloud_api_key" {
 variable "prefix" {
   description = "A unique identifier for resources. Must begin with a lowercase letter and end with a lowerccase letter or number. This prefix will be prepended to any resources provisioned by this template. Prefixes must be 16 or fewer characters."
   type        = string
+  default     = ""
```
[the remark this answers is not in the document]
Comment (Nikhil22February) [reply solutions/quickstart/variables.tf]: Removed default values from prefix and region.

File: solutions/quickstart/variables.tf (hunk 2 above)
Comment (Nikhil22February) [reply solutions/quickstart/variables.tf:266]: I created this new branch out of standard_quickstart and it was throwing error conflict. 
Reverted back the changes and copied the override_json_string values from init_quickstart branch.

**Comment by Nikhil22February** [comment]
Readme file got changed from Pketaki from her custom branch and reflecting in this branch, created a new PR link. https://github.com/terraform-ibm-modules/terraform-ibm-zvsi/pull/160
